src/devices/schraeder.c

```diff
@@ -84,6 +84,80 @@ static int schraeder_callback(bitbuffer_t *bitbuffer) {
 	return 0;
 }
 
+ /**
+  * TPMS Model: Schrader Electronics EG53MA4
+  * Contributed by: Leonardo Hamada (hkazu)
+  * Probable packet payload:
+  * 
+  * SSSSSSSSSS ???????? IIIIII TT PP CC
+  *
+  * S = sync
+  * ? = might contain the preamble, status and battery flags
+  * I = id (24 bits), could extend into flag bits (?)
+  * P = pressure, 25 mbar per bit
+  * T = temperature, degrees Fahrenheit
+  * C = checksum, sum of byte data modulo 256
+  *
+  */
+  
+static int schrader_EG53MA4_callback(bitbuffer_t *bitbuffer) {
+	char time_str[LOCAL_TIME_BUFLEN];
+	data_t *data;
+	uint8_t b[10];
+	uint32_t serial_id;
+	char id_str[9];
+	uint flags;
+	char flags_str[9];
+	int pressure;		// mbar
+	int temperature;	// Fahrenheit
+	int checksum;
+
+	/* Reject wrong amount of bits */
+	if ( bitbuffer->bits_per_row[0] != 120)
+		return 0;
+		
+	/* shift the buffer 40 bits to remove the sync bits */
+	bitbuffer_extract_bytes(bitbuffer, 0, 40, b, 80);
+
+	/* Calculate the checksum */
+	checksum = (b[0]+b[1]+b[2]+b[3]+b[4]+b[5]+b[6]+b[7]+b[8]) & 0xff; 
+    if (checksum != b[9]) {
+        return 0;
+    }
+
+	local_time_str(0, time_str);
+
+	/* Get serial number id */
+	serial_id = (b[4] << 16) | (b[5] << 8) | b[6];
+	sprintf(id_str, "%06X", serial_id);
+	
+	flags = (b[0] << 24) | (b[1] << 16) | (b[2] << 8) | b[3];
+	sprintf(flags_str, "%08x", flags);
+
+	pressure = b[7] * 25;
+	temperature = b[8];
+
+	if (debug_output >= 1) {
+		fprintf(stderr, "Schrader EG53MA4 TPMS decoder\n");
+		bitbuffer_print(bitbuffer);
+		fprintf(stderr, "id = 0x%X\n", serial_id);
+		fprintf(stderr, "CHECKSUM = %x\n", checksum);
+	}
+
+	data = data_make("time", "", DATA_STRING, time_str,
+					"model", "", DATA_STRING, "Schrader Electronics EG53MA4",
+					"type", "", DATA_STRING, "TPMS",
+					"flags", "", DATA_STRING, flags_str,
+ 					"id", "ID", DATA_STRING, id_str,
+					"pressure_bar",  "Pressure",    DATA_FORMAT, "%.03f bar", DATA_DOUBLE, ((double)pressure)/1000.0,
+					"temperature_C", "Temperature", DATA_FORMAT, "%.0f C", DATA_DOUBLE, fahrenheit2celsius((double)temperature),
```

Comment:
don't convert, output native native units and use the `-C si` command line switch